server/proxy/pf_context.c

```diff
@@ -129,7 +139,7 @@ static BOOL client_to_proxy_context_new(freerdp_peer* client, rdpContext* ctx)
 	obj->fnObjectEquals = ChannelId_Compare;
 
 	obj = HashTable_ValueObject(context->channelsByFrontId);
-	obj->fnObjectFree = StaticChannelContext_free;
+	obj->fnObjectFree = HashStaticChannelContext_free;
```

Comment:
Nit: The name now reads to me as "hash of (a) static channel context", but this doesn't seem to be the case here. Maybe there is a different way to name this (I am actually fine with the previous `StaticChannelContext_free` (or the same written in snake case))?

Reply:
hard to name it, as it only exists due to `C` lack of overloaded parameter support.